viewer/packages/rendering/src/CadMaterialManager.ts

```diff
@@ -262,3 +269,25 @@ function applyToModelMaterials(materials: Materials, callback: (material: THREE.
   callback(materials.triangleMesh);
   callback(materials.simple);
 }
+
+function isRawShaderMaterial(material: THREE.Material): material is THREE.RawShaderMaterial {
```

Comment:
Is it recent that we started using `RawShaderMaterial`?

Reply:
Yes, we just introduced this.